ui/component/or-attribute-history/src/index.ts

```diff
@@ -381,90 +396,136 @@ export class OrAttributeHistory extends translate(i18next)(LitElement) {
-                                pointBackgroundColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-color"),
-                                pointRadius: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-radius")),
-                                pointBorderWidth: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-border-width")),
-                                pointHoverBackgroundColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-color"),
-                                pointHoverBorderColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-border-color"),
-                                pointHoverRadius: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-radius")),
-                                pointHoverBorderWidth: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-border-width")),
-                                pointHitRadius: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hit-radius")),
-                                fill: false
+                this._chartOptions = {
+                    animation: false,
+                    grid: {
+                        show: true,
+                        backgroundColor: this._style.getPropertyValue("--internal-or-asset-viewer-panel-color"),
+                        borderColor: this._style.getPropertyValue("--internal-or-attribute-history-text-color"),
+                        left: 15,
+                        right: 15,
+                        containLabel: true
+                    },
+                    backgroundColor: this._style.getPropertyValue("--internal-or-asset-viewer-panel-color"),
+                    tooltip: {
+                        trigger: 'axis',
+                        axisPointer: { type: 'cross'},
+                        formatter: (params: any) => {
+                            if (Array.isArray(params) && params.length > 0) {
+                                const yValue = params[0].value[1];
+                                return yValue !== undefined ? yValue.toString() : '';
+                            }
+                            return ''
                             }
-                        ]
                     },
-                    options: {
-                        responsive: true,
-                        maintainAspectRatio: false,
-                        onResize:() => this.dispatchEvent(new OrAttributeHistoryEvent('resize')),
-                        plugins: {
-                            legend: {
-                                display: false
+                    toolbox: {
+                        right: '2%',
+                        top: '5%',
+                        feature: {
+                            dataView: {readOnly: true},
```

Comment:
Can we update the data view, so that "timestamp" and "value" texts are shown in the first row?
I saw that you can copy paste this into MS Excel, which is nice 👍